app/requirements.txt

```diff
@@ -0,0 +1,7 @@
+cffi==1.17.1
+cryptography==44.0.2
+pwinput==1.0.3
```

Comment:
why do you need pwinput for GUI app?

P.S. we have separate requirements file for key generator utility